test/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+  File "/home/robin/Desktop/dev_ayon/dev/ayon-cpp-api/ayon_automator/AyonCiCd/Project.py", line 53, in run
+    self.func(*args, **kwargs)
+  File "/home/robin/Desktop/dev_ayon/dev/ayon-cpp-api/AyonBuild.py", line 154, in startTestServer
+    from test import TestServer
+ImportError: cannot import name 'TestServer' from 'test' (/usr/lib/python3.10/test/__init__.py)
+"""
```

Comment:
I had to add this `__init__.py` file there to prevent the issue above.

Reply:
yes lets doo that. 
having the doc string in there might help us in the future nice idea.

Reply:
Feel free to remove the docstring there, I put it there mostly to share the error message with you.